upload/src/addons/SV/WarningImprovements/XF/Admin/Controller/Warning.php

```diff
@@ -80,7 +98,9 @@ public function actionSave(ParameterBag $params)
     public function actionSort()
     {
         $categoryRepo = $this->getCategoryRepo();
-        $categories = $categoryRepo->findCategoryList()->fetch();
+        $categories = $categoryRepo->findCategoryList()
+            ->where('warning_count', '<>', 0)
```

Comment:
Should include all categories in the admin-cp side, even if empty. Find to filter them out on the front-end

Reply:
Done